src/openms/source/ANALYSIS/OPENSWATH/IonMobilityScoring.cpp

```diff
@@ -70,16 +71,13 @@ namespace OpenMS
 
    @param profile The ion mobility data
    @param im_grid The grid to be used
-   @param al_int_values The intensity vector (y)
-   @param al_im_values The ion mobility vector (x)
    @param eps Epsilon used for computing the ion mobility grid
    @param max_peak_idx The grid position of the maximum
 
   */
-  void IonMobilityScoring::alignToGrid_(const IonMobilogram& profile,
+  void IonMobilityScoring::alignToGrid_(const Mobilogram& profile,
```

Comment:
mm, I'm not sure. From my understanding, the `MobilityPeak1D` structure is for a single data point, and the `Mobilogram` structure is to hold every `MobilityPeak1D` data point, to represent a 1D mobilogram.

The `alignToGrid_` method is for aligning a vector of `Mobilogram`'s for *N* transitions.

If we want to move it to `Mobilogram`, I think we may need to modify the structure to assume that it would house a vector of `Mobilograms`.

Reply:
Ok then we can keep the method here then. I thought it was aligning to a common grid however if it is just aligning mobilograms might be more OpenSwath specific then